.github/workflows/generate-builds.yml

```diff
@@ -46,7 +46,7 @@ jobs:
     - name: Generate soh.otr
       run: |
         export PATH="/usr/lib/ccache:/usr/local/opt/ccache/libexec:$PATH"
-        cmake --no-warn-unused-cli -H. -Bbuild-cmake -GNinja -DCMAKE_BUILD_TYPE:STRING=Release
+        cmake --no-warn-unused-cli -H. -Bbuild-cmake -GNinja -DCMAKE_BUILD_TYPE:STRING=Release -DBUILD_REMOTE_CONTROL=1
```

Comment:
Does the "Generate soh.otr" step need to build with remote support? It doesn't even fully build soh, it only runs the otr target.

Reply:
Good call